core/src/main/resources/hudson/tasks/BatchFile/help-unstableReturn.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    If set, the batch errorlevel result that will be interpreted as an unstable build result.
+    In WinNT4 and beyond, the ERRORLEVEL is stored as a Long - that is, a four byte, signed integer;
+    yielding maximum and minimum values of 2147483647 and -2147483648, respectively.
```

Comment:
I tested a simple batch file with a single line: 'exit /b -2147483648' and the errorlevel is set to '-2147483648'. It works on win7, etc. I think it's better not to limit the available range, although hardly used.